Base/QTGUI/Resources/UI/qSlicerSettingsModulesPanel.ui

```diff
@@ -282,14 +235,14 @@
       </sizepolicy>
      </property>
      <property name="text">
-      <string>Additional module paths:</string>
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Additional module paths:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;(drag-and-drop&lt;br/&gt;files or folders)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Was this on purpose?

Reply:
While I was editing the module settings panel, I've added the “(drag-and-drop files or folders here)" text to the additional module paths list (because users may not know without that that they can drag files there). I've used italics formatting for the note and that's why the text was switched from plain text to html. It is an unrelated change, so if you think it is better to split it out into a separate commit then I can do that.

Reply:
I see.  Sounds like a good addition, but it looked like it might have been a mistake.  I think it's fine to leave it in this PR.  No reason to make extra work for this small thing.